### DISCUSSION TITLE
Wiki.JS doesn't work with a preexisting repo

### DISCUSSION THREAD

**Discussion by Zombiehamser** [requarks/wiki#3348] [Help / Questions]
**Question**
Hellow! I have the same problem like in ticket https://github.com/Requarks/wiki/issues/507 but I don't know where I can find ./repo of Wiki.JS? I try search all "repo" from host machine to docker container, and find nothing.

**Host Info (please complete the following information):**
OS: [e.g. RedOS Linux (RHEL base distr), Docker]
Wiki.js version: [e.g. 2.0.12]
Database engine: [e.g. postgreSQL 11.5]

**Comment by NGPixel** [comment]
This folder doesn't exist by default. It's created when you sync git the first time.

**Comment by Zombiehamser** [comment]
> This folder doesn't exist by default. It's created when you sync git the first time.

Maybe I dont have this .repo folder yet, but I almost have problem with git sync "refusing to merge unrelated histories", Its talking about non-empty repo at wikiJS server, as I understand. 
![2019-12-09 14_22_09-AdminOpera](https://user-images.githubusercontent.com/57707453/70432064-6ed01e80-1a8f-11ea-843e-0691fe0d29bb.jpg)

**Comment by ssimontis** [comment]
The Git sync is designed to work with an empty repo. Can you create a brand new repo and try again? @Zombiehamser

**Comment by NGPixel** [comment]
Delete the repo folder on disk and try again.

**Comment by kexic** [comment]
I added the Git repository as a storage solution long after the wiki.js project has been in motion.  There's a lot of data in that local repo.  I just need the UI to run the following commands in order to force push its git repo into that space.

git remote add origin git@github.com:COMPANY/WIKI.git
git push -u origin master

If I were to delete my local repository, wouldn't I loose my entire active wiki?!?  If necessary, add a --force to the push.  I really just need the Git repo to act as a Backup Storage Device.

**Comment by NGPixel** [comment]
No, content is stored in the DB, not in the local git repo.
